lib/salad_ui/input.ex

```diff
@@ -12,7 +12,7 @@ defmodule SaladUI.Input do
       <.input type="password" placeholder="Enter your password" />
   """
   attr :id, :any, default: nil
-  attr :name, :any, default: nil
+  attr :name, :any
```

Comment:
Like `value` we don't want a default so that `assign_new` will call the fun if the user doesn't supply a value for `name`.